examples/app.R

```diff
@@ -10,6 +10,7 @@ shinyApp(
     shiny.info::info_panel(
       shiny.info::git_info(),
       shiny.info::powered_by("Appsilon", link = "appsilon.com"),
+      shiny.info::inspect_btn_ui(),
```

Comment:
As this example grows bigger, can we for now keep all information visible and put some model, or just info on the top about: `to show info press Ctrl+Shift+K`